hal/src/main/native/mau/AnalogAccumulator.cpp

```diff
@@ -0,0 +1,121 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "HAL/AnalogAccumulator.h"
+
+#include "AnalogInternal.h"
+#include <VMXIO.h>
+#include <VMXErrors.h>
+#include <VMXResource.h>
+#include "VMXPointers.h"
+
+using namespace hal;
+
+namespace hal {
+    namespace init {
+        void InitializeAnalogAccumulator() {}
+    }
+}
+
+extern "C" {
+    HAL_Bool HAL_IsAccumulatorChannel(HAL_AnalogInputHandle analogPortHandle, int32_t* status) {
+        auto port = analogInputHandles->Get(analogPortHandle);
+        if (port == nullptr) {
+            *status = HAL_HANDLE_ERROR;
+            return false;
+        }
+        for (int32_t i = 0; i < kNumAccumulators; i++) {
```

Comment:
Where is "kNumAccumulators" defined?   In VMX-pi, the number of accumulator (resources) is equal 4 (equal to the number of analog input channels.

Reply:
In the WPI they're defined in the PortsInternal.h file. In it "constexpr int32_t kNumAccumulators = 2;" but I'm planning on importing the numbers and setting them from VMX-pi